akka-remote/src/main/scala/akka/remote/artery/BufferPool.scala

```diff
@@ -73,19 +73,18 @@ private[remote] object EnvelopeBuffer {
 
   val VersionOffset = 0 // Byte
   val FlagsOffset = 1 // Byte
-  // 2 bytes free // TODO re-align values to not have this empty space
-  val UidOffset = 4 // Long
-  val SerializerOffset = 12 // Int
+  val UidOffset = 2 // Long
```

Comment:
um, these are unaligned fields. I would prefer to have everything at 4 or 8 byte boundaries